src/main/java/com/facebook/bitEncoderDecoder/bitwise/BitwiseEncoderImpl.java

```diff
@@ -51,4 +51,23 @@ private String checkIfEven(String triplet) {
         return result.toString();
     }
 
+    private String[] getTriplets(String input) {
+        int numberOfTriplets = (input.length() / 3) + 1;
+        int lastPlace = numberOfTriplets - 1;
+        if (input.isBlank()) {
```

Comment:
input.isBlank() should be at the begining of method, before any operations on input